frontend/src/features/notes/list/NoteListView.tsx

```diff
@@ -31,16 +36,25 @@ export const NoteListView: React.FunctionComponent<INoteListViewProps> = (
   const [sort, setSort] = React.useState<TableSort<Api_Note>>({});
   const [noteResult, setNoteResult] = React.useState<Api_Note[]>([]);
 
-  React.useEffect(() => {
+  const [isAddNotesOpened, openAddNotes, closeAddNotes] = useModalManagement();
+  const [isViewNotesOpened, openViewNotes, closeViewNotes] = useModalManagement();
+
+  const fetchNotes = useCallback(async () => {
```

Comment:
Q: Could the fetching of the notes have been done on this view's container?

Reply:
This is the container 😄

Reply:
Maybe we need to talk on Friday to stop naming containers "Views"